core/src/main/java/org/fao/geonet/kernel/unpublish/UnpublishInvalidMetadataJob.java

```diff
@@ -77,10 +77,7 @@
 import org.springframework.scheduling.quartz.QuartzJobBean;
 
 import java.sql.SQLException;
-import java.util.Date;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
I am personally used to configure my idea with "editor/code style/java/imports/class count to use import with '*'" = 50 or more.

Reply:
good to know, thanks :)